Objects/frameobject.c

```diff
@@ -1146,7 +1145,7 @@ frame_init_get_vars(_PyInterpreterFrame *frame)
     int offset = PyUnstable_Code_GetFirstFree(co);
     for (int i = 0; i < co->co_nfreevars; ++i) {
         PyObject *o = PyTuple_GET_ITEM(closure, i);
-        frame->localsplus[offset + i] = Py_NewRef(o);
+        frame->localsplus[offset + i] = PyStackRef_NewRef(PyStackRef_StealRef(o));
```

Comment:
I think all the `PyStackRef_NewRef(PyStackRef_StealRef())` double calls should be:

```
frame->localsplus[offset + i] = _PyStackRef_NewRefDeferred(o);
```